src/main/java/life/qbic/model/petab/Time.java

```diff
@@ -0,0 +1,15 @@
+package life.qbic.model.petab;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Time {
+  @JsonProperty
+  String unit;
+
+  @Override
+  public String toString() {
+    return "Time{" +
+        "unit='" + unit + '\'' +
+        '}';
+  }
```

Comment:
Shouldn't this also have a value, or is this only the unit specification?

Reply:
one would think so, but the example yaml only has unit